include/pmacc/memory/buffers/HostBufferIntern.hpp

```diff
@@ -124,12 +124,12 @@ class HostBufferIntern : public HostBuffer<TYPE, DIM>
     void setValue(const TYPE& value)
     {
         __startOperation(ITask::TASK_HOST);
-        size_t current_size = this->getCurrentSize();
+        long current_size = static_cast< long >(this->getCurrentSize());
         auto memBox = getDataBox();
         typedef DataBoxDim1Access<DataBoxType > D1Box;
         D1Box d1Box(memBox, this->getDataSpace());
         #pragma omp parallel for
-        for (size_t i = 0; i < current_size; i++)
+        for (long i = 0; i < current_size; i++)
```

Comment:
QUESTION:
I am a bit confused. Before this pull request a `size_t` type number `current_size` was compared to a `size_t` type number `i`. 
How could this cause an error of comparing unsigned and signed integers if both are of the same type `size_t` (unsigned).

Is just the last pull request #2505 not shown here - since the omp parallel should not use size_t, as you pointed put in #2505.  But I see a `size_t` as old code.

Reply:
*offline discussion with @psychocoderHPC:*
Just update your branch to include your own commit. That prevents this merge conflict error.

Reply:
Yeah, sorry, I branched out of the older version of mainline/dev.

Reply:
sure, just rebase against the current `dev`